node/src/chain/header_validation.rs

```diff
@@ -18,10 +19,13 @@ use node_data::message::payload::RatificationResult;
 use node_data::message::ConsensusHeader;
 use node_data::{ledger, StepName};
 use std::sync::Arc;
+use std::time::{SystemTime, UNIX_EPOCH};
 use thiserror::Error;
 use tokio::sync::RwLock;
 use tracing::info;
 
+const MARGIN_TIMESTAMP: u64 = 10;
```

Comment:
Didn't we agree on having `MARGIN_TIMESTAMP = 3`?

Reply:
Correct